### PR TITLE
Fixing issues with the new userflow where a missing mqttTo-config causes NPEs

### PR DESCRIPTION
**Pull request by codepitbull**
**Motivation**

Resolves #27907

**Changes**
When introducing the new userflow (first creating the config, then the tags + mappings) the changes didn't reflect through to all config objects.
Several places had NotNull-annotations and others relied on the values not being null.

### CODE REVIEW

File: hivemq-edge/src/main/java/com/hivemq/api/resources/impl/ProtocolAdaptersResourceImpl.java [hunk 1]
```diff
@@ -576,4 +577,61 @@ public Response getTagSchema(String protocolId) {
...
+                    objectMapper.convertValue(adapter.getAdapter(), new TypeReference<>() {
+                    });
+
+            final List<Map<String, Object>> domainTags = adapter.getDomainTagModels()
+                    .stream()
+                    .map(dtm -> DomainTag.fromDomainTagEntity(dtm, adapterId))
+                    .map(DomainTag::toTagMap)
+                    .collect(Collectors.toList());
+
+            System.out.println(config);
```
Comment (DC2-DanielKrueger) [inline hivemq-edge/src/main/java/com/hivemq/api/resources/impl/ProtocolAdaptersResourceImpl.java:618]: remove the souts pls

File: hivemq-edge/src/main/java/com/hivemq/api/resources/impl/ProtocolAdaptersResourceImpl.java [hunk 2]
```diff
@@ -576,4 +577,61 @@ public Response getTagSchema(String protocolId) {
...
+            System.out.println(domainTags);
+            protocolAdapterManager.addAdapter(adapterType, adapterId, config, domainTags);
+        } catch (final IllegalArgumentException e) {
+            if (e.getCause() instanceof UnrecognizedPropertyException) {
+                e.printStackTrace();
+                ApiErrorUtils.addValidationError(errorMessages,
+                        ((UnrecognizedPropertyException) e.getCause()).getPropertyName(),
+                        "Unknown field on adapter configuration");
+            }
+            e.printStackTrace();
```
Comment (DC2-DanielKrueger) [inline hivemq-edge/src/main/java/com/hivemq/api/resources/impl/ProtocolAdaptersResourceImpl.java:628]: pls look for any souts and e.printStackTraces in the code

File: modules/hivemq-edge-module-file/src/main/java/com/hivemq/edge/adapters/file/config/FileToMqttConfig.java [hunk 3]
```diff
@@ -65,6 +65,7 @@ public int getMaxPollingErrorsBeforeRemoval() {
     }
 
     public @NotNull List<FileToMqttMapping> getMappings() {
+        System.out.println("MAPPING2 " + mappings);
```
Comment (DC2-DanielKrueger) [inline modules/hivemq-edge-module-file/src/main/java/com/hivemq/edge/adapters/file/config/FileToMqttConfig.java:68]: Souts

**Review by DC2-DanielKrueger — APPROVED** [review]

File: modules/hivemq-edge-module-file/src/main/java/com/hivemq/edge/adapters/file/config/FileToMqttConfig.java (hunk 3 above)
Comment (codepitbull) [reply modules/hivemq-edge-module-file/src/main/java/com/hivemq/edge/adapters/file/config/FileToMqttConfig.java]: done

File: hivemq-edge/src/main/java/com/hivemq/api/resources/impl/ProtocolAdaptersResourceImpl.java (hunk 2 above)
Comment (codepitbull) [reply hivemq-edge/src/main/java/com/hivemq/api/resources/impl/ProtocolAdaptersResourceImpl.java]: done

**Pull request merged**